ionoscloud/utils.go

```diff
@@ -46,17 +46,21 @@ func resourceIpFailoverImporter(ctx context.Context, d *schema.ResourceData, met
 		return nil, err
 	}
 
-	if lan.Properties.IpFailover != nil {
-		err := d.Set("ip", *(*lan.Properties.IpFailover)[0].Ip)
-		if err != nil {
-			return nil, err
-		}
-	}
+	failoverSlice := lan.Properties.IpFailover
+	if lan.Properties != nil && lan.Properties.IpFailover != nil && len(*failoverSlice) > 0 {
```

Comment:
why don't you verify here failoverSlice != nil instead of lan.Properties.IpFailover != nil?